streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamTransformValuesTest.java

```diff
@@ -54,92 +44,6 @@ public class KStreamTransformValuesTest {
     private final Properties props = StreamsTestUtils.getStreamsConfig(Serdes.Integer(), Serdes.Integer());
     private InternalProcessorContext context = mock(InternalProcessorContext.class);
 
-    @SuppressWarnings("deprecation") // Old PAPI. Needs to be migrated.
```

Comment:
Do we have a corresponding test for `process(FixedKeyProcessor)` -- the comment says "needs to be migrated -- did this migration already happen?